.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -24,13 +24,13 @@ body:
         **NOTE: If your version is NOT listed then please upgrade before opening the GH issue. Thank you**
       options:
         # renovate: datasource=github-tags depName=cilium/cilium
-        - 'v1.16.0-rc.1 or higher'
+        - 'higher than v1.16.0-rc.1 and lower than v1.16.99'
```

Comment:
We don't use the versions `vX.Y.99` any more, this new version seems more confusing that than the existing version. Should this just say `lower than v1.17.0`? (Same for below)

Reply:
Oh, good idea. Let's try with that.